File: BloggerDomain/Post.cs
using System;
using System.Collections.Generic;

namespace BloggerDomain
{
    public class Post : BaseEntity
    {
        public byte[] Image { get; set; }
        public string Text { get; set; }
        public DateTime CreatedAt { get; set; }
        public Guid PersonId { get; set; }
        public Person Person { get; set; }
        public ICollection<Like> Likes { get; set; }
        public ICollection<Comment> Comments { get; set; }
    }
}

File: BloggerClient/ClientApp/src/app/models/post.ts
import { Like } from "./like";
import { Comment } from "./comment";

export class Post {
  id: string;
  createdAt: Date;
  image: any;
  text: string;
  personName: string;
  personImage: any;
  personId: string;
  likesCount: number;
  commentsCount: number;
  currentUser: boolean;
}
